### DISCUSSION TITLE
New to GoReleaser

### DISCUSSION THREAD

**Discussion by seanpclayton** [goreleaser/goreleaser#4557] [Q&A] [answered]
I have been running my workflow and I get stuck on the same step every time where I get an error that the current folder is invalid and not a git repository. I have tried adding `git init` to the workflow and have eliminated the `fetch-depth` action. I'm at a loss here. The repository is initialized and all of the files are present in the root directory including .goreleaser.yaml. What am I missing?
![Screenshot 2024-01-18 at 12 10 58](https://github.com/goreleaser/goreleaser/assets/61811605/31d3ce13-c8c9-44ab-932f-48ba12f067e0)
![image](https://github.com/goreleaser/goreleaser/assets/61811605/020283ab-ece7-42f3-97ba-17e257776b6a)

**Comment by caarlos0** [comment, marked as answer]
there are many examples you can look up to in the goreleaser org https://github.com/orgs/goreleaser/repositories?q=example&type=all&language=&sort=


other than that, I can't help based on error screenshots only.

**Reply by seanpclayton** [reply]
Thank you!
